lib/solidus_dev_support/sandbox.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+# Used in the sandbox rake task in Rakefile
+
+set -e
+
+case "$DB" in
+postgres|postgresql)
+  RAILSDB="postgresql"
+  ;;
+mysql)
+  RAILSDB="mysql"
+  ;;
+sqlite|'')
+  RAILSDB="sqlite3"
+  ;;
+*)
+  echo "Invalid DB specified: $DB"
+  exit 1
+  ;;
+esac
+
+rm -rf ./sandbox
+bundle exec rails new sandbox --database="$RAILSDB" \
```

Comment:
Is there a reason we are not simply using `common:test_app` or `extension:test_app` with a custom path?

Reply:
Yes, the reason is twofold: on one side we've found out it wasn't so straightforward, on the other this is the exact same approach used by the main solidus repo. The last one was also the reason for using this naming (vs. for example _dev_app_), having the same pattern for the whole solidus ecosystem should help reducing the surprise effect and probably will also help spreading the knowledge about this feature.